tools/generate-doc-preview-index.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+
+HEAD_COMMIT_SHA=$(git rev-parse HEAD)
+cd $(git rev-parse --show-toplevel)/target/doc
+
+{
+    echo '<!doctype html>'
+    echo '<html>'
+    echo '<head>'
+    echo '  <metadata charset="utf-8">'
+    echo "  <title>Doc preview for ${HEAD_COMMIT_SHA}</title>"
```

Comment:
A more human-readable and useful title would be something like `git log -1 --oneline`.

Reply:
I like it! Thanks for the suggestion.